taqueria-plugin-core/clean.ts

```diff
@@ -0,0 +1,45 @@
+import { execCmd, getCurrentEnvironmentConfig, sendAsyncErr, sendJsonRes } from '@taqueria/node-sdk';
+import { RequestArgs } from '@taqueria/node-sdk/types';
+
+const ECAD_FLEXTESA_IMAGE = 'ghcr.io/ecadlabs/taqueria-flextesa';
+const FLEXTESA_IMAGE = 'oxheadalpha/flextesa';
+const LIGO_IMAGE = 'ligolang/ligo';
+const ARCHETYPE_IMAGE = 'completium/archetype';
+const ECAD_TZKT_IMAGE = 'alirezahaghshenas/tzkt';
+
+const getDockerImageIdsCmd = (): string => {
+	const images = [ECAD_FLEXTESA_IMAGE, FLEXTESA_IMAGE, LIGO_IMAGE, ARCHETYPE_IMAGE, ECAD_TZKT_IMAGE];
+	const imageFilters = images.reduce((acc, image) => `${acc} --filter "reference=${image}"`, '');
+	const cmd = `docker images --quiet ${imageFilters} | uniq`;
+	return cmd;
+};
+
+const removeStates = () =>
+	execCmd('rm .taq/*state*.json')
+		.catch(() => Promise.reject(new Error('No state files exist in the .taq/ folder')));
```

Comment:
Does this mean the task will error if there is no state file?

Reply:
Yep! But the taq command wouldn't give an error because this is caught later on, so that if you do `echo $?` after the taq command, you get a success status.

Reply:
But if this fails, so will the any tasks that are called after it (the removal of images)

Reply:
hmm yeah true. Will change that

Reply:
I mean, those will be silently skipped, am I right?

Reply:
Yeah fixed!